lig3dlens/alignment.py

```diff
@@ -1,6 +1,6 @@
 import multiprocessing
 from pathlib import Path
-from typing import Optional
+from typing import Optional, Dict, Generator, Tuple
```

Comment:
This line is causing the CI to fail, we can either manually correct ... or running `isort` will fix this for you; 

```
$ isort --profile black lig3dlens tests setup.py
```